_submissions/mod-chat-censure

```diff
@@ -0,0 +1,19 @@
+Name: Chat Censure
+Author: Veridisquo
+Repository: https://github.com/xDevICCI/module_chat_censure
+Download: https://github.com/poszer/module_chat_censure/archive/master.zip
```

Comment:
same here

Reply:
i fixed, i used same models description of poszer